troubleshooting/troubleshooting-android-push-notifications-stopped-working.md

```diff
@@ -0,0 +1,64 @@
+---
+title: Push Notifications to Android stopped working
+description: Check your Google API key if you receive the error "Error authenticating to Google Cloud Messaging"
```

Comment:
Describe only the error, not the solution. E.g.:
Push notifications throw "Error authenticating to Google Cloud Messaging"